src/cmds/restic/cmd_find.go

```diff
@@ -84,7 +85,95 @@ func parseTime(str string) (time.Time, error) {
 	return time.Time{}, errors.Fatalf("unable to parse time: %q", str)
 }
 
-func findInTree(repo *repository.Repository, pat findPattern, id restic.ID, prefix string, snapshotID *string) error {
+type findNode restic.Node
+type omit *struct{}
```

Comment:
You do not need this, simply use a small data type like `byte` or or `string` instead of `omit`.

I think the generic way to declare something like your `omit` type would be the empty interface (`interface{}`), a pointer to the empty struct is rather unusual ;)